invenio_jobs/models.py

```diff
@@ -44,12 +46,17 @@ class Job(db.Model, Timestamp):
     default_args = db.Column(JSON, default=lambda: dict(), nullable=True)
     schedule = db.Column(JSON, nullable=True)
 
-    # TODO: See if we move this to an API class
     @property
     def last_run(self):
         """Last run of the job."""
         return self.runs.order_by(Run.created.desc()).first()
 
+    def last_run_with_status(self, status):
```

Comment:
it assumes `status` is something that can be added to the query. Is this OK? 

for github we have something similar but the status can be `None`:

```python
q = self.releases if status is None else self.releases.filter_by(status=status)
```

Reply:
Yes, I should add some checks for this one. It's either that or making `last_run` a method, but I want something that can be easily used e.g. in a template.